apps/fis/fis-ds-update-web/aat.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: fis-ds-update-web
+  namespace: fis
+spec:
+  values:
+    nodejs:
+      ingressSessionAffinity:
+        enabled: true
```

Comment:
why is this needed?

Reply:
i have removed it already in latest commit